docs/source/02_get_started/04_configuration.md

```diff
@@ -30,7 +30,7 @@ domain: "do.qhub.dev" # top level URL exposure to monitor JupyterLab
     Azure, and `local` for a local or existing kubernetes deployment.
 
  - `ci_cd`: is the continuous integration and continuous deployment
-   framework to use. Currently only `github-actions` is supported.
+   framework to use. Currently, only `github-actions` is supported.
 
  - `domain`: is the top level URL to put JupyterLab and future
    services under such a monitoring. For example `jupyter.qhub.dev`
```

Comment:
I know that this is a general domain thing, even though shouldn't we remove the jupyter for this too?

Reply:
Yes, we should. Thank for flagging that. Just fixed now.